### PR TITLE
Small usability changes plus mobile friendly formatting

### PR DESCRIPTION
**Pull request by sujancho**
**Mobile Friendly Layout adjustments**
1. Cranny Buy prices collapse to a 2x3 block on mobile
2. Daisy Mae First time toggle condenses lightly on mobile

**Turnip Table Changes**
1. Wrapped turnip table in a scrollable view with light shadowing for affordance
2. Moved Week Max/Min to front for easy view
3. Adjust turnip table spacing to be even
4. Adjusted table spacing and header typography

**Misc Changes**
1. Added Nook's Cranny header + additional dividers
2. Hid Arrow buttons from input fields

### CODE REVIEW

File: js/scripts.js [hunk 1]
```diff
@@ -95,14 +95,14 @@ const calculateOutput = function (data, first_buy) {
...
     var out_line = "<tr><td>" + poss.pattern_description + "</td>"
+    out_line += `<td class="one">${poss.weekMin}</td><td class="one">${poss.weekMax}</td>`;
     for (let day of poss.prices.slice(1)) {
       if (day.min !== day.max) {
         out_line += `<td>${day.min}..${day.max}</td>`;
       } else {
         out_line += `<td class="one">${day.min}</td>`;
       }
-    }
-    out_line += `<td class="one">${poss.weekMin}</td><td class="one">${poss.weekMax}</td></tr>`;
```
Comment (alexwh) [inline js/scripts.js:105]: the `</tr>` never gets added back in

**Comment by sujancho** [comment]
turns out my changes aren't working on mobile (although they work in developer tools) going to take another stab at this and submit as requests since i know theres a ui overhaul in possible review.

**Comment by theRTC204** [comment]
You were missing the viewport meta tag that tell the browser how to scale elements.

**Comment by sujancho** [comment]
AH! Thanks, I'm not going to resubmit now since the new ui overhaul is in.

**Pull request closed without merging**